docker/test-server/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM python:3.9.0-alpine
+
+ARG ETESYNC_VERSION
+
+ENV PIP_DISABLE_PIP_VERSION_CHECK=1
+ENV PIP_NO_CACHE_DIR=1
+
+# install packages and pip requirements first, in a single step,
+COPY /requirements.txt /requirements.txt
+RUN set -ex ;\
+    apk add libpq postgresql-dev --virtual .build-deps coreutils gcc libc-dev libffi-dev make ;\
+    pip install -U pip ;\
+    pip install --no-cache-dir --progress-bar off -r /requirements.txt ;\
+    apk del .build-deps make gcc coreutils ;\
+    rm -rf /root/.cache
+
+COPY . /app
+
+RUN set -ex ;\
+    mkdir -p /data/static /data/media ;\
+    cd /app ;\
+    mkdir -p /etc/etebase-server ;\
+    cp docker/test-server/etebase-server.ini /etc/etebase-server ;\
+    sed -e '/ETEBASE_CREATE_USER_FUNC/ s/^#*/#/' -i /app/etebase_server/settings.py ;\
+    chmod +x docker/test-server/entrypoint.sh
+
+RUN set -ex ;\
+    cd /app ;\
+    python manage.py migrate
```

Comment:
You can't run migrate in the Dockerfile. Migrate needs to be called from the entrypoint because it needs to be called every time you run the server and it needs to call on the production server. So move this line to entrypoint.sh.

Reply:
Why is this?  The idea is that the test image starts up already having the DB in place, so that it can start up quickly.  The docker image never carries data between container invocations.

Reply:
You are right. I guess that's a special case for the test server. :)
In the production server you want to do it in runtime so that the db used in production is updated.

It still feels odd, so maybe add a comment, but otherwise it's fine by me.